specification/Specification.adoc

```diff
@@ -0,0 +1,63 @@
+= 3D Tiles Specification
+Cesium GS, Inc
+v1.1.0
+:toc2:
+:toclevels: 2
+:sectnums:
+:source-highlighter: rouge
+:rouge-style: github
+
+image:figures/3DTiles_dark_color_small.png[]
+
+This document describes the specification for 3D Tiles, an open standard for streaming massive heterogeneous 3D geospatial datasets.
```

Comment:
It's unfortunate that the logo and intro text is split off into this separate document since it leaves `README.md` looking a little bare. But I can understand why it's done this way.

Reply:
When I started the conversion, I did a few experiments regarding the https://docs.asciidoctor.org/asciidoc/latest/document/header/ in AsciiDoc. There are some options and degrees of freedom, and I assume that for a document that is about to be submitted for standardization, there will be some sort of "title page template" (and we'll have to see whether this will just be "prefixed" to the PDF, or has to be integrated into the PDF that we generate).

To avoid the "bare" `README.adoc`, we could do something with 
```
ifdef::env-github[]
// include some image here
endif::[]
ifndef::env-github[]
// Don't :-) 
endif::[]
```

Reply:
Ah good idea!